sormas-backend/src/test/java/de/symeda/sormas/backend/event/EventParticipantFacadeEjbTest.java

```diff
@@ -551,6 +552,28 @@ public void searchEventParticipantsByPersonOtherDetail() {
 		eventParticipantIndexDtos = getEventParticipantFacade().getIndexList(eventParticipantCriteria, 0, 100, null);
 		assertEquals(0, eventParticipantIndexDtos.size());
 	}
+	@Test
+	public void testSaveEventParticipantWithEventArchiving() {
+		TestDataCreator.RDCF rdcf = creator.createRDCF("Region", "District", "Community", "Facility");
+		UserDto user = createUser(rdcf);
+		EventDto event = createEvent(user, rdcf);
+		// Validate that for a newly created Event Participant the archived status is kept from the Event
+		getEventFacade().archive(Collections.singletonList(event.getUuid()));
+		PersonDto eventPersonArchived = creator.createPerson("EventArchived", "OrganizerArchived");
+		EventParticipantDto archivedEventParticipantDto = creator.createEventParticipant(event.toReference(), eventPersonArchived, "event Director", user.toReference());
+		assertTrue(getEventParticipantFacade().isArchived(archivedEventParticipantDto.getUuid()));
+
+		// Validate that for a newly created Event Participant the active status is kept from the Event
+		getEventFacade().dearchive(Collections.singletonList(event.getUuid()), "reason");
+		PersonDto eventPersonActive = creator.createPerson("EventActive", "OrganizerActive");
+		EventParticipantDto activeEventParticipantDto = creator.createEventParticipant(event.toReference(), eventPersonActive, "event Director", user.toReference());
+		assertFalse(getEventParticipantFacade().isArchived(activeEventParticipantDto.getUuid()));
+
+		//Validate that for an already existing Event Participant the previous status is kept
```

Comment:
This test description does not match: Creating a new event participant does not relate to what happened to the `archivedEventParticipantDto`

The last `assertEquals(getEventParticipantFacade().isArchived(archivedEventParticipantDto.getUuid()), getEventParticipantFacade().isArchived(alreadyArchivedParticipantDto.getUuid()));` does not check the expected behaviour but only if two values in the database are the same (but not if true or false).